packages/transformers/babel/src/babelErrorUtils.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import type {BaseAsset} from '@parcel/types';
+
+export type BabelError = Error & {
+  loc?: {
+    line: number,
+    column: number,
+    ...
+  },
+  source?: string,
+  ...
+};
+
+export async function babelErrorEnhancer(
+  error: BabelError,
+  asset: BaseAsset
+): Promise<BabelError> {
+  if (error.loc) {
+    error.message = error.message
+      .slice(asset.filePath.length + 1, error.message.indexOf('\n'))
+      .trim();
+  }
+
+  error.source = (await asset.getCode()).toString();
```

Comment:
Kinda wanted to add it into the transformerrunner. But that would cause unexpected behaviour for things that import multiple assets without Parcel involvement like css does...

Reply:
`getCode` should already return a string I think?

Reply:
I thought there was a typeerror saying it returned buffer, I'll double check

Reply:
Yeah... Not entirely sure why I put it there there was a flow error originally and fixed it quickly by adding that...